docs/050-breaking-changes.rst

```diff
@@ -272,6 +273,83 @@ Syntax
   ``override``, ``partial``, ``promise``, ``reference``, ``sealed``,
   ``sizeof``, ``supports``, ``typedef`` and ``unchecked``.
 
+.. _interoperability:
+
+Interoperability with older Contracts
+=====================================
+
+It is still possible to interface with contracts written for Solidity versions prior to
```

Comment:
Also the other way around.

Reply:
True - it's unlikely to be needed, though, isn't it? I'll just add it in parenthesis here for now.